astro/src/content/docs/lifecycle/manage-users/verification/identity-verification-using-email.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Identity Pre-Verification Using Email
+description: Learn how to verify email identities before user creation.
+navcategory: developer
+section: lifecycle
+subcategory: manage users
+tertcategory: verification
+---
+import Aside from 'src/components/Aside.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import PremiumPlanBlurb from 'src/content/docs/_shared/_premium-plan-blurb.astro';
+
+## Overview
+
+As of version 1.62.0, FusionAuth can be configured to pre-verify users' identities, in the registration process, before user creation.
+
+To enable pre-verification, do the following:
+
+- Configure the Tenant Email Server
+- Enable verification for the identity type(s) on the tenant
+- Create an advanced registration form with the verification step
+- Enable self-service registration in your app
+
+Let's step through each of these, but first, there are some required prerequisites.
```

Comment:
```suggestion
```
[suggestion]: IMO this goes without saying, and the following 'prerequisites' header is enough to introduce prerequisites.